pom.xml

```diff
@@ -44,15 +44,14 @@
 
     <issueManagement>
         <system>GitHub</system>
-        <url>https://github.com/janinko/ghprb/issues</url>
+        <url>https://github.com/jenkinsci/ghprb/issues</url>
```

Comment:
no

Reply:
There are 168 open in the first, 92 in your replacement. Really JIRA should be used and both closed, but this is an issue for the plugin maintainer to sort out.

Reply:
yes, "-plugin" is missing